controllers/v1/userExtension.js

```diff
@@ -158,11 +156,33 @@ module.exports = class userExtension extends Abstract {
 
   /**
    * @api {get} /admin-service/api/v1/userExtension/create 
-   * to create user 
+   * To create the user 
    * @apiVersion 1.0.0
    * @apiGroup User Creation
```

Comment:
@rakeshSgr - For all API endpoints the grouping can be **Users**

Reply:
resolved